camunda-formio-tasklist-vue/src/components/mixins/TaskListMixin.vue

```diff
@@ -7,6 +7,7 @@ import BaseMixin from './BaseMixin.vue';
 })
 export default class TaskListMixin extends Mixins(BaseMixin) {
 
-  public maximize = true;
+  public maximize: boolean = true;
+  public userName: string = '';
```

Comment:
Any reason why it is set up to empty string and not null?

Reply:
No particular reason, should we make it null @krishnan-aot ?